src/v/redpanda/admin/api-doc/partition.json

```diff
@@ -135,6 +135,40 @@
                     ]
                 }
             ]
+        },
```

Comment:
```
admin_api: Add new request for get transactions for partition
Fixes #2987
```

I think that you should only add `Fixes ...` to a commit that resolves the issue. In this case it is not this commit which resolves that issue, but rather the entire PR. So putting that annotation in the PR cover letter is more appropriate.